src/Reader/Extension/Atom/Entry.php

```diff
@@ -545,10 +545,11 @@ public function getSource()
      * Attempt to absolutise the URI, i.e. if a relative URI apply the
      *  xml:base value as a prefix to turn into an absolute URI.
      *
-     * @param  string $link
-     * @return string
+     * @param string $link
+     *
+     * @return null|string
      */
-    protected function absolutiseUri($link)
+    protected function absolutiseUri($link): ?string
```

Comment:
Please do not add php typehints as this will is a BC break unless the class is final.